src/Compilers/CSharp/Portable/Syntax/StackAllocArrayCreationExpressionSyntax.cs

```diff
@@ -7,9 +7,13 @@ namespace Microsoft.CodeAnalysis.CSharp.Syntax
     public partial class StackAllocArrayCreationExpressionSyntax
```

Comment:
I see that the stackalloc syntax derives from the local declaration syntax. Could you add a corresponding parsing test (`using` and stackalloc)? And also take a note to do a full test on that scenario (not just parsing).

Can you or @agocke start a test plan issue for this feature work, if you don't have one already? It is useful to collect test ideas.